### PR TITLE
Fix adding additional Disc 2 to certain multi-disc games

### PR DESCRIPTION
**Pull request by RisingFog**

### CODE REVIEW

**Comment by lioncash** [comment]
wxStrings have a CmpNoCase function you can use for case insensitive comparison btw.

File: Source/Core/DolphinWX/GameListCtrl.cpp [hunk 1]
```diff
@@ -415,7 +415,7 @@ void CGameListCtrl::InsertItemInReportView(long _Index)
 	if (gameini.GetIfExists("EmuState", "Title", &title))
 		name = title;
 
-	if (rISOFile.IsDiscTwo() && name.find("Disc 2") == std::string::npos)
+	if (rISOFile.IsDiscTwo() && (name.Lower().find("disc 2") == std::string::npos && name.Lower().find("disc2") == std::string::npos))
```
Comment (JosJuice) [inline Source/Core/DolphinWX/GameListCtrl.cpp]: Why are there parentheses around the disc 2 and disc2 checks combined? Having one set of parentheses around each check would make more sense, but having parentheses might not be necessary at all.

Comment (RisingFog) [inline Source/Core/DolphinWX/GameListCtrl.cpp]: forgot to remove those as I originally did an or comparison instead of and.

**Comment by JosJuice** [comment]
Seems like a good idea, but I'll refrain from commenting "LGTM" since I don't have any games to test this with.

**Comment by BhaaLseN** [comment]
In C#, I'd recommend using a case insensitive comparison instead of doing ```toLower```, but I have no idea what the C++ equivalent would be.
Can you name a few games where those cases would happen (both the non "Disc" spelling with different case and the "Disc2" without-a-space variant)?

**Comment by degasus** [comment]
Or shall we look at the new regex module in c++11? http://www.cplusplus.com/reference/regex/

**Comment by JosJuice** [comment]
@BhaaLseN https://github.com/dolphin-emu/dolphin/pull/2006#issuecomment-105925987 and https://dolphin-emu.googlecode.com/issues/attachment?aid=68520014000&name=3.PNG&token=ABZ6GAf0mvJMY85c_FlNu8D4ij3rJfZ4BQ%3A1432740711200&id=6852&mod_ts_token=ABZ6GAdp3aqJqSJo72CwM_Ymdk_8dB2IPw%3A1432740711200&inline=1

**Comment by tapcio** [comment]
Works as expected but I like to see "(Disc 1)" for first disc, because most multidisc games that have "Disc 2" also have "Disc 1" if first disc title.

**Comment by RisingFog** [comment]
@tapcio There really isn't an easy way to do that, it was discussed in the original PR

**Comment by JosJuice** [comment]
Adding "(Disc 1)" to the first disc of multi-disc games is more complicated, because it would require us to scan through the game list to see if there is a second discs so that all single-disc games don't show up as "(Disc 1)". The primary reason for adding "(Disc 2)" was to be able to see which disc is which, and that works well enough without having a "(Disc 1)" marker.

**Comment by tapcio** [comment]
Maybe I want to much, nevermind :P

PS: Maybe you can use GameSetting ini to rename game title?

**Comment by RisingFog** [comment]
@tapcio much more work than needed for a simple UI change

**Comment by RisingFog** [comment]
@degasus wxString doesn't have regex_match, and wxRegEx isn't enabled by default in Dolphin

**Pull request merged**